seed/lib/xml_mapping/mapper.py

```diff
@@ -50,49 +47,68 @@ def default_buildingsync_profile_mappings():
         # from BAE
         "F"
     ]
+    return valid_units
 
-    mapping = BASE_MAPPING_V2.copy()
-    base_path = mapping['property']['xpath'].rstrip('/')
-    result = []
-    for col_name, col_info in mapping['property']['properties'].items():
-        from_units = col_info.get('units')
-        if from_units not in valid_units:
-            from_units = None
-
-        sub_path = col_info['xpath'].replace('./', '')
-        absolute_xpath = f'{base_path}/{sub_path}'
-        result.append({
-            'from_field': absolute_xpath,
-            'from_field_value': col_info['value'],
-            'from_units': from_units,
-            'to_field': col_name,
-            'to_table_name': 'PropertyState'
-        })
 
-    # BAE results
+def get_BAE_mappings():
```

Comment:
can you update the method name to be `get_bae_mappings` (all lower case)? It's silly, just want to keep things consistent.

Reply:
done!